app/controllers/placement_needs_controller.rb

```diff
@@ -0,0 +1,33 @@
+class PlacementNeedsController < ApplicationController
+  def new
+    @child = Child.find(params[:child_id])
+    @placement_need = PlacementNeed.new
+    authorize @placement_need
+  end
+
+  def create
+    @placement_need = PlacementNeed.new(
+      placement_need_params.merge(child_id: params[:child_id]),
+    )
+    authorize @placement_need
+
+    if @placement_need.save
+      redirect_to shortlist_path(params[:child_id])
+    else
+      render "new"
+    end
+  end
+
+private
+
+  def placement_need_params
+    params.require(:placement_need).permit(
+      :child_id,
+      :criteria,
+      :placement_date,
+      :postcode,
+      :location_radius,
+      array: PlacementNeed::OPTIONS,
```

Comment:
What is this for?

Reply:
it is for the OPTIONS, but we don't need it if we are going only with a single placement criteria